newnew.properties

```diff
@@ -0,0 +1,5 @@
+#Wed Feb 01 16:33:04 CET 2023
```

Comment:
Is this file needed?

Reply:
whoops, they are not supposed to be there